WordPress/gutenberg
# Port global styles code to `lib/compat/wordpress-5.9`: front assets

This PR continues porting global styles code into `lib/compat/wordpress-5.9` folder. It moves the code responsible to enqueue the global styles assets in the frontend.

## How to test

- Activate TwentyTwentyTwo.
- Load the front end of the site.
- The expected result is that the theme styles are properly loaded (or modifications done via the site editor's global styles sidebar, if any).
